packages/fast-tooling-react/src/css-editor/editor.tsx

```diff
@@ -33,225 +27,52 @@ export default class CSSEditor extends Foundation<
         managedClasses: void 0,
     };
 
-    private spacingPropertyNames: string[] = [
-        SpacingProperty.marginBottom,
-        SpacingProperty.marginLeft,
-        SpacingProperty.marginRight,
-        SpacingProperty.marginTop,
-        SpacingProperty.paddingBottom,
-        SpacingProperty.paddingLeft,
-        SpacingProperty.paddingRight,
-        SpacingProperty.paddingTop,
-    ];
-
-    private postionPropertyNames: string[] = [
-        "position",
-        Location.left,
-        Location.right,
-        Location.top,
-        Location.bottom,
-    ];
-
-    private heightPropertyNames: string[] = ["height"];
-
-    private widthPropertyNames: string[] = ["width"];
-
-    private backgroundPropertyNames: string[] = ["background"];
+    /**
+     * The custom control plugins used in the form
+     */
+    private controlPlugins: StandardControlPlugin[];
 
-    private colorPropertyNames: string[] = ["color"];
+    constructor(props: CSSEditorProps) {
+        super(props);
 
-    private borderPropertyNames: string[] = ["borderColor", "borderStyle", "borderWidth"];
+        const exampleData: any = getDataFromSchema(cssEditorSchema);
 
-    private boxShadowPropertyNames: string[] = ["boxShadow"];
+        this.state = {
+            data: exampleData,
+        };
 
-    private borderRadiusPropertyNames: string[] = [
-        "borderRadius",
-        "borderTopLeftRadius",
-        "borderTopRightRadius",
-        "borderBottomRightRadius",
-        "borderBottomLeftRadius",
-    ];
+        this.controlPlugins = [
+            new StandardControlPlugin({
+                id: "color",
+                control: (config: ControlConfig): React.ReactNode => {
+                    return <CSSColor data={this.props.data} onChange={config.onChange} />;
+                },
+            }),
+        ];
+    }
 
     public render(): React.ReactNode {
         return (
             <div className={this.props.managedClasses.cssEditor}>
-                {this.renderPosition()}
-            </div>
-        );
-    }
-
-    private renderPosition(): React.ReactNode {
-        return (
-            <React.Fragment>
-                <CSSSpacing
-                    jssStyleSheet={{ cssSpacing: { marginBottom: "10px" } }}
-                    data={this.getSpacingData()}
-                    onChange={this.handleCSSUpdate(CSSComponent.spacing)}
-                />
-                <CSSPosition
-                    data={this.getPositionData()}
-                    onChange={this.handleCSSUpdate(CSSComponent.position)}
-                />
-                <CSSWidth
-                    data={this.getWidthData()}
-                    onChange={this.handleCSSUpdate(CSSComponent.width)}
+                <Form
+                    className={this.props.managedClasses.cssEditor_form}
```

Comment:
I would not add the ability to do style overrides to the form just yet as we don't know what shape this will take when we update the Form.